src/components/pages/AdminHome/AdminCourseCard.js

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import { useHistory } from 'react-router-dom';
+import { Card } from 'antd';
+
+// Using AntD to pull in a card component to show course details. The course info comes from the parent component AdminCourses with course passed through props. Ability to go directly to course details or course edit from the card page.
+export default function CourseCard(props) {
+  const { course } = props;
+  const history = useHistory();
+
+  const handleEditCourse = () => {
+    history.push(`/admin-edit-course/${course.course_id}`);
+  };
+
+  const viewCourseDetails = id => {
+    history.push(`/admin-course-details/${course.course_id}`);
+  };
+
+  return (
+    <div className="admin-course-card-container">
+      <Card title={course.course_name}>
+        <h3>
```

Comment:
Could we use `.map` for these `<h3>` headers? Maybe it would be a little cleaner.

Reply:
we're gonna pull it from props

Reply:
I believe it makes more sense not to in this case. Each "course" has a lot of extra details and we don't need all of them for this card. I like the idea for clean code, but to me it doesn't make much sense (course is also an object so map won't work). I could definitely be wrong but personally I think it makes more sense to pick and choose which fields to display